src/main/resources/reference.conf

```diff
@@ -31,6 +31,24 @@ cassandra-journal {
   # Delay between connection retries
   connect-retry-delay = 5s
 
+  # Cassandra connection connections per host core local
+  connections-per-host-core-local = 1
+
+  # Cassandra connection connections per host max local
```

Comment:
the documentation of these properties are very dense, perhaps link to the cassandra driver connection pool documentation?
perhaps place related properties in a separate sub-section, e.g. `connection-pool` ?